HISTORY.rst

```diff
@@ -2,6 +2,63 @@
 History
 =======
 
+2.9.0 (2022-07-13)
+------------------
+
+* Dropped support for Python 3.6 and 3.7.
+* Added support for Python 3.9 and 3.10.
+* Fix an issue where SI prefixes of the ``degC`` units would give incorrect
+  values in conversions. See `PR #176
+  <https://github.com/yt-project/unyt/pull/176>`_. Thank you to Lee Johnston
+  (@l-johnston on GitHub) for the contribution.
+* Fix an issue when using ``matplotlib_support``, plot an empty unyt array,
+  would result in an error when changing units. See `PR #180
+  <https://github.com/yt-project/unyt/pull/180>`_. Thank you to Josh Borrow
+  (@JBorrow on GitHub) for the contribution.
+* Fix an issue where units would be printed twice in formatted strings with
+  an ``unyt_array`` embedded. See `PR #188
+  <https://github.com/yt-project/unyt/pull/180>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Add a method to parse a ``unyt_quantity`` from a string expression. See `PR #191
+  <https://github.com/yt-project/unyt/pull/191>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Fix an issue where a ``unyt_array`` with dtype int8 could not be converted
+  to a different unit. See `PR #197
+  <https://github.com/yt-project/unyt/pull/197>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* The import time for ``unyt`` has been optimized. See `PR #199
```

Comment:
Modesty++
Also this particular PR should probably be mentioned for the actual change that's performed. It's still (remotely) possible that it's breaking to someone.
```suggestion
* The import time for ``unyt`` has been reduced. See `PR #199
```